pkg/controllers/manager.go

```diff
@@ -119,16 +151,74 @@ func (m *GenericControllerManager) RegisterControllers(ctx context.Context, cont
 }
 
 // workerIndex func returns a list of worker addresses from a workflow.
-func wokerIndexFunc(obj client.Object) []string {
+func workflowWorkerAddrIndexFunc(obj client.Object) []string {
```

Comment:
comment needs updating to match func name

Reply:
fixed